modules/claims_api/spec/fixtures/v2/veterans/claims/claims_by_participant_id_response.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
I wonder why this file is showing up as new.  i added it as part of my work.  🤔

Reply:
my bad, for some reason I thought I renamed it, but looks like I just duplicated and forgot to delete. will do 👍